great_expectations/execution_engine/split_and_sample/sqlalchemy_data_splitter.py

```diff
@@ -693,11 +695,11 @@ def get_split_query_for_data_for_batch_identifiers_for_split_on_mod_integer(
         column_name: str,
         mod: int,
     ) -> Selectable:
-        """Divide the values in the named column by `divisor`, and split on that"""
-        # query = f"SELECT DISTINCT(\"{self.column_name}\" / {divisor}) AS my_var FROM {self.table_name}"
+        """Divide the values in the named column by `mod`, and split on that"""
+        # query = f"SELECT DISTINCT(\"{self.column_name}\" % {mod}) AS my_var FROM {self.table_name}"
```

Comment:
Can this be deleted? Is there a reason it is commented out?

Reply:
seconded

Reply:
I don't see why not, a lot of these methods were moved as-is from the execution engine and can use some clean up.

Reply:
@cdkini I believe that this and similar comments were added by @anthonyburdi  -- I fixed the typos in a couple of them, but am in no position to delete them.  Thanks!

Reply:
Done all that (had to make some fixes to the present PR).